src/mito2/src/sst/parquet/reader.rs

```diff
@@ -467,6 +501,9 @@ impl Drop for ParquetReader {
         READ_ROW_GROUPS_TOTAL
             .with_label_values(&["min_max_index_selected"])
             .inc_by(self.metrics.num_row_groups_min_max_selected as u64);
+        PRECISE_FILTER_ROWS_TOTAL
+            .with_label_values(&["parquet"])
+            .inc_by(self.metrics.num_rows_precise_filtered as u64);
```

Comment:
where do you set `self.metrics.num_rows_precise_filtered`?

Reply:
Good catch... 🫣